main.js

```diff
@@ -134,9 +134,12 @@ function disableSaveButton(event){
 function saveOutfit() {
   var outfitName = saveOutfitInput.value;
   var savedOutfitCard = `<button class="button-style">${outfitName}<img class="close-btn" src="assets/close.svg" alt="Close"></button>`;
+  newOutfit.title = outfitName;
   savedOutfitsList.insertAdjacentHTML('beforeend', savedOutfitCard);
   window.localStorage.setItem(outfitName, savedOutfitCard);
+  outfits.push(newOutfit);
   clearInputField(saveOutfit);
+  resetDataModel();
```

Comment:
Adding the title to the object before saving card. Then firing the resetDataModel function.

Reply:
awesome. looks really clean